lang/en.json

```diff
@@ -1,5 +1,7 @@
 {
    "TagManager": {
+      "AddThisTagDescription": "Add the AddThis social media buttons and more to your website.",
+      "AddThisTagHelp": "TODO",
```

Comment:
should we just not define a help text? eg in the tag we can add:

```
public function getHelp()
    {
return '';
}
```

Reply:
I am not sure what we could write as a good help text for AddThis.

Reply:
That's why I meant we could just leave it empty. The description is already pretty clear.

Reply:
Makes sense. Done in https://github.com/matomo-org/tag-manager/pull/59/commits/bd9ecec86d6fb60ae1ba075c10bb82408922eab7